pinnwand/template/layout.html

```diff
@@ -6,6 +6,7 @@
     <link rel="stylesheet" href="/static/pygments.css" type="text/css">
     <link rel="shortcut icon" type="image/x-icon" href="/static/favicon.ico">
     <script type="text/javascript" src="/static/pinnwand.js"></script>
+    <script type="text/javascript" src="/static/indent.js"></script>
```

Comment:
I'd like all javascript to be served from a single file so you can add your code to ``pinnwand.js`` directly.

Reply:
Sure. I will update once, i get this cleaned